scanner/redhatbase.go

```diff
@@ -19,8 +20,22 @@ import (
 // https://github.com/serverspec/specinfra/blob/master/lib/specinfra/helper/detect_os/redhat.rb
 func detectRedhat(c config.ServerInfo) (bool, osTypeInterface) {
 	if r := exec(c, "ls /etc/fedora-release", noSudo); r.isSuccess() {
-		logging.Log.Warnf("Fedora not tested yet: %s", r)
-		return true, &unknown{}
+		if r := exec(c, "cat /etc/fedora-release", noSudo); r.isSuccess() {
+			re := regexp.MustCompile(`\d+`)
+			release := re.FindString(r.Stdout)
+			ver, err := strconv.Atoi(release)
+			if err != nil {
+				logging.Log.Warnf("Failed to parse Fedora version: %s", release)
+				return true, &unknown{}
+			}
+			if ver < 32 {
+				logging.Log.Warnf("Versions prior to Fedora 32 are not supported, detected version is %s", release)
+				return true, &unknown{}
```

Comment:
If the scan target has `/etc/fedora-release`, it is assumed to be Fedora.
```suggestion
				return true, newFedora(c)
```